<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VolumeResultDialogForm</class>
 <widget class="QDialog" name="VolumeResultDialogForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>536</width>
    <height>252</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Volume Result</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTableWidget" name="m_resultTableWidget">
     <column>
      <property name="text">
       <string>POLYGON_ID</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>PROFILE(m)</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>EMBANKMENT(m)</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>AREA(m)</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>QUOTA IDEAL</string>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout_3">
     <item row="0" column="1">
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="0" column="2">
      <widget class="QPushButton" name="m_savePushButton">
       <property name="text">
        <string>&amp;Save</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="te::qt::widgets::HelpPushButton" name="m_helpPushButton">
       <property name="text">
        <string>Help</string>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QPushButton" name="m_cancelPushButton">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>te::qt::widgets::HelpPushButton</class>
   <extends>QPushButton</extends>
   <header>terralib/qt/widgets/help/HelpPushButton.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
